insights/parsers/tests/test_ethtool.py

```diff
@@ -565,3 +570,5 @@ def test_ethtool(self):
     def test_ethtool_fail(self):
         with pytest.raises(ParseException):
             ethtool.Ethtool(context_wrap(FAIL_ETHTOOL_A_1, path="ethtool_eth1"))
+        with pytest.raises(ParseException):
```

Comment:
Since there are different ParseExceptions in the parser, better check them correspondingly by check the exception message.
And, would you mind refactor the unit test to pytest style in this commit? (unittest2pytest would help you do that easily)

Reply:
Sure.